DataManagementSystem/scripts/dirac-dms-find-lfns.py

```diff
@@ -62,7 +62,6 @@
   if not result['OK']:
     gLogger.error( 'Can not access File Catalog:', result['Message'] )
     DIRAC.exit( -1 )
-  lfnList = result['Value']
+  lfnList = sorted( result['Value'] )
 
-  for lfn in lfnList:
-    print lfn
+  gLogger.notice( '\n'.join( lfn for lfn in lfnList ) )
```

Comment:
Sorry, I didn't remember this...
B.t.w. why was the comment removed that you had put in that PR? Why not fix gLogger ;-)?